setup.md

```diff
@@ -2,4 +2,54 @@
 title: Setup
 ---
 
-#FIXME
+## Setup Python
+
+Make sure that you have the latest Python installed. The latest Python released is `Python 3.11.3`
+
+- To check the installation open a terminal and run:
+
+```bash
+python –version
+```
+- or if you have Python3:
+
+```bash
+python3 –version
+```
+
```

Comment:
I'd refer to installation instructions at https://swcarpentry.github.io/python-novice-inflammation/#install-python instead of providing installation instructions here.